src/app/app.component.ts

```diff
@@ -74,6 +79,13 @@ export class AppComponent implements OnInit {
       resolveSelectedValue: this.sampleResolve
     });
 
+    this.translate.addLangs(['en', 'fr']);
+    this.translate.setDefaultLang('en');
+    this.fetchMockedTranslationsData().then((translationsData: any) => {
```

Comment:
Do we really want this here? This might be useful for testing, but doesn't seem like it should windo up here.